source/server/connection_handler_impl.h

```diff
@@ -364,11 +384,12 @@ class ConnectionHandlerImpl : public Network::ConnectionHandler,
   UnitFloat listener_reject_fraction_{UnitFloat::min()};
 };
 
-class ActiveUdpListenerBase : public ConnectionHandlerImpl::ActiveListenerImplBase,
+using ActiveListenerImplBase = ConnectionHandlerImpl::ActiveListenerImplBase;
```

Comment:
I think it's easier to read without this `using`

Reply:
ok. removed